tests/TestRvsdgs.hpp

```diff
@@ -618,110 +603,92 @@ class IndirectCallTest1 final : public RvsdgTest {
 class IndirectCallTest2 final : public RvsdgTest {
 public:
   [[nodiscard]] jlm::llvm::delta::node &
-  GetDeltaG1() const noexcept
-  {
+  GetDeltaG1() const noexcept {
```

Comment:
These following changes are not following the coding standard.
Or is it different for different cases?